ember-toucan-core/src/components/form/controls/input.ts

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+
+interface ToucanFormControlsInputComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    isDisabled?: boolean;
+    onChange?: (value: string, e: Event | InputEvent) => void;
```

Comment:
I added a re-usable type here if you want to give it a go! 

```suggestion
    onChange?: OnChangeCallback<string>;
```

(Reference: https://github.com/CrowdStrike/ember-toucan-core/pull/36/files#diff-90b0ef8397a4c8dabf3349619492464098ba317ce1542a255dcc35ac8640615f)